prov/util/src/util_fabric.c

```diff
@@ -61,7 +61,7 @@ int ofi_fabric_init(const struct fi_provider *prov,
 	atomic_initialize(&fabric->ref, 0);
 	dlist_init(&fabric->domain_list);
 	fastlock_init(&fabric->lock);
-	fabric->name = prov_attr->name;
+	fabric->name = user_attr->name;
```

Comment:
Thanks for catching this. User could free user_attr->name. I'll add an strdup() here.